Dockerfile.assisted-service-build

```diff
@@ -11,9 +11,8 @@ USER 0
 
 RUN mkdir build && chmod g+xw -R build/
 
-RUN yum install -y epel-release \
-    && yum install -y --setopt=skip_missing_names_on_install=False \
-        gcc genisoimage jq git libvirt-client libvirt-devel java
+RUN yum install -y --setopt=skip_missing_names_on_install=False \
+        gcc genisoimage git libvirt-client libvirt-devel java
```

Comment:
While you are at it
```suggestion
        gcc genisoimage git libvirt-client libvirt-devel java && yum -y clean all  && rm -rf /var/cache
```

Reply:
it's a backport, so I'm leaning towards keeping it the same as the upstream change